src/NodaTime/TimeZones/Resolvers.cs

```diff
@@ -67,6 +67,19 @@ public static class Resolvers
         };
 
         /// <summary>
+        /// A <see cref="SkippedTimeResolver"/> which shifts values in the "gap" forward by the duration
+        /// of the gap (which is usually 1 hour).
```

Comment:
Perhaps add a "remarks" that this means it returns a value corresponding to the instant that *would* have occurred using the earlier zone interval, if there hadn't been a transition? That might lead to a simpler implementation, too - not sure. (It could be that I'm mistaken about the effect...)